test/unit/vl-map-draw-point-action.test.html

```diff
@@ -0,0 +1,29 @@
+<!doctype html>
+<html>
+
+<head>
+  <meta charset="utf-8">
+  <script src="../../../@webcomponents/webcomponentsjs/webcomponents-lite.js"></script>
+  <script src="../../../web-component-tester/browser.js"></script>
+
+  <script type="module" src="../../src/vl-map-all.js"></script>
+  <script src="/node_modules/sinon/pkg/sinon.js"></script>
+</head>
+
+<body>
+  <script type="module">
+    import {VlMapDrawPointAction} from '../../src/vl-map-draw-point-action.js';
+
+    suite('vl-map-draw-action', () => {
+      setup((done) => {
+        customElements.whenDefined('vl-map-draw-point-action').then(() => done());
+      });
+    
+      test('een punt teken actie is een map actie', () => {
+        assert.isTrue(VlMapDrawPointAction.isVlMapAction());
+      });
```

Comment:
moet ge hier ook nie checken of onderliggend de "correcte" actie wordt aangemaakt?

Reply:
Aangezien de logica in vl-map-draw-point-action zich beperkt tot het aanmaken van een vl-mapactions actie en de logica van het toevoegen en activeren van de actie zich in de vl-map-action class zit, beperkt de unit test zich hier tot het controleren van de extend.

In dit geval is er meer aandacht besteed aan de e2e testen aangezien de meerwaarde op unit niveau beperkt blijft door het samenspel met de achterliggende vl-mapactions en OL dependencies.